traffic_ops/app/db/reencrypt/reencrypt.go

```diff
@@ -0,0 +1,406 @@
+package main
```

Comment:
commands should document their usage in the `main` package's GoDoc, for an example, check out [the GoDoc for `godoc` itself](https://github.com/golang/tools/blob/v0.1.2/cmd/godoc/doc.go).